ingestor/core/pubsub.py

```
@@ -0,0 +1,4 @@
+import json
+
+async def publish_event(rdb, channel, event):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)